package.json

```diff
@@ -23,11 +23,12 @@
     "lint:styles": "stylelint 'src/**/*.js'",
     "parity": "parity --config parity-config.toml",
     "precommit": "lint-staged",
-    "prep": "mkdir build/ && mkdir build/embed && cp src/index.html build/ && mkdir build/assets/ && cp -r src/assets/* build/assets/",
+    "prep": "mkdir build/ && mkdir build/embed && cp src/index.html build/ && mkdir build/assets/ && cp -r src/assets/* build/assets/ && yarn run prep:clappr",
+    "prep:clappr": "cp -r node_modules/clappr/dist/* build/",
```

Comment:
If we don't do this, then on our `embed/` routes clappr tries to load a  font from our server that doesn't exist; the request pends forever and the browser appears to be loading infinitely even though the player is playing.

Solution came from [this clappr issue](https://github.com/clappr/clappr/issues/1153#issuecomment-249427459)